docker_supp.sh

```diff
@@ -0,0 +1,24 @@
+echo "About to start conda update, this may take some time..."
+source setup/setup_conda.sh Linux-x86_64
+source setup/setup.sh
+conda clean -t
+conda clean -p
+
+if [ -d "webapp/www/" ]; then
+    cp /index.html webapp/www/index.html
+fi
+
+if [ -d "/conf" ]; then
+    echo "Found configuration, overriding..."
+    cp -r /conf/* conf/
+fi
```

Comment:
Ok, I'm not sure what I thought I was leaving this here for anymore. The correct thing to do is just delete it?

Reply:
well, you still need to configure the settings somehow. If you just delete it, how will we configure the webapp vs. analysis images for example? Again,

> Did you test the resulting image with the multi-tier-cronjob, for example?

You need to update that in general because otherwise the scheduled tasks are not actually using the new images.
Presumably we will also need to refer to the new images differently since they are no longer published on dockerhub

Reply:
In general, we don't plan to store the server specific configuration in the `conf` files but in a GitHub repo that only had configs. So we shouldn't plan to have separate `conf` files for each production install.

In any case, we should figure all that out as part of https://github.com/e-mission/e-mission-docs/issues/790